frame/bounties/src/lib.rs

```diff
@@ -185,55 +156,53 @@ pub enum BountyStatus<AccountId, BlockNumber> {
 	},
 }
 
-// Note :: For backward compatibility reasons,
-// pallet-bounties uses Treasury for storage.
-// This is temporary solution, soon will get replaced with
-// Own storage identifier.
-decl_storage! {
-	trait Store for Module<T: Config> as Treasury {
-
-		/// Number of bounty proposals that have been made.
-		pub BountyCount get(fn bounty_count): BountyIndex;
+#[frame_support::pallet]
+pub mod pallet{
```

Comment:
`cargo fmt` might complain if it did not already:

```suggestion
pub mod pallet {
```

Reply:
ah yes, I should run that locally anyways